modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/collectors/RuntimeStatisticCollector.java

```diff
@@ -95,13 +112,69 @@ public static void init() {
                     eventConsumerTime;
             log.info("Statistics Entry Expiration time set to " + eventExpireTime + " milliseconds");
             new MediationFlowController();
+
+            if (isOpenTracingEnabled) {
+                initOpenTracing(isCollectingPayloads, isCollectingProperties);
+            }
         } else {
             if (log.isDebugEnabled()) {
                 log.debug("Statistics is not enabled in \'synapse.properties\' file.");
             }
         }
     }
 
+    private static void initOpenTracing(boolean isCollectingPayloads, boolean isCollectingProperties) {
+        final String DEFAULT_JAEGER_SAMPLER_MANAGER_HOST = "localhost";
+        final String DEFAULT_JAEGER_SAMPLER_MANAGER_PORT = "5778";
+        final String DEFAULT_JAEGER_SENDER_AGENT_HOST = "localhost";
+        final String DEFAULT_JAEGER_SENDER_AGENT_PORT = "6831";
+        final String DEFAULT_JAEGER_REPORTER_MAX_QUEUE_SIZE = "100";
+        final String DEFAULT_JAEGER_REPORTER_FLUSH_INTERVAL = "10000";
+
+        // Jaeger Sampler Configurations
+        String samplerManagerHost =
+                SynapsePropertiesLoader.getPropertyValue(
+                        StatisticsConstants.JAEGER_SAMPLER_MANAGER_HOST, DEFAULT_JAEGER_SAMPLER_MANAGER_HOST);
+        String samplerManagerPort =
+                SynapsePropertiesLoader.getPropertyValue(
+                        StatisticsConstants.JAEGER_SAMPLER_MANAGER_PORT, DEFAULT_JAEGER_SAMPLER_MANAGER_PORT);
+        String samplerManagerHostPort = samplerManagerHost + ":" + samplerManagerPort;
```

Comment:
Do we need to read host and port separately? I think user can provide the entire URL that we build here programmatically

Reply:
As discussed offline, we will be reading the host and port separately.
Since the user can't configure the protocol, asking `host` and `port` is better instead of `host:port`.